addons/website/static/src/builder/plugins/options/card_image_alignment_option.js

```diff
@@ -14,14 +14,21 @@ export class CardImageAlignmentOption extends BaseOptionComponent {
         super.setup();
         this.state = useDomState(async (editingElement) => {
             await this.waitForAllImageloaded(this.env.getEditingElements());
-            const imageToWrapperRatio = this.getImageToWrapperRatio(editingElement);
-            const hasCoverImage = !!editingElement.querySelector(".o_card_img_wrapper");
+            const coverImageWrapperEl = editingElement.querySelector(
+                ":scope > .o_card_img_wrapper"
+            );
+            const hasCoverImage = !!coverImageWrapperEl;
+            const imageToWrapperRatio = hasCoverImage
+                ? this.getImageToWrapperRatio(coverImageWrapperEl)
+                : null;
+            const hasShape = hasCoverImage
+                ? !!coverImageWrapperEl.querySelector(".o_card_img[data-shape]")
+                : false;
```

Comment:
@AlessandroLupo Can you help me understand what exactly these lines fix? I couldn’t see the before and after of this change.